.github/workflows/cd.yml

```diff
@@ -0,0 +1,44 @@
+name: Dependabot CD to GCP
+
+on:
+  push:
+    branches:
+      - master
```

Comment:
please configure this workflow to run when the ci workflow is successful after a push to the master branch. see docs https://docs.github.com/en/actions/reference/events-that-trigger-workflows#workflow_run